lib/qbwc/job.rb

```diff
@@ -83,7 +96,7 @@ def next
 
   def reset
     self.request_index = 0
-    self.requests = []
+    #self.requests = []
```

Comment:
My intention is that the requests are generated only at the start of the job and are then iterated through. It shouldn't regenerate the requests at the start of each iteration.